bin/fillTestData/Messages.js

```diff
@@ -17,17 +36,25 @@ var addDays = function addDays(date, days) {
   return result;
 };
 
-var addThreads = function (numThreads, maxMessages) {
+var addThreads = function () {
   var index = 0;
+  var numThreads = argv._[0];
+  var maxMessages= argv._[1];
```

Comment:
I believe in Yargs these can be grabbed with `positional` args: http://yargs.js.org/docs/#api-positionalkey-opt rather than the accessing internal object.

Reply:
Thanks!  I was looking for that.